src/app/shared/components/todo-list/todo-list.component.ts

```diff
@@ -1,18 +1,18 @@
 import {Component, EventEmitter, Input, Output} from '@angular/core';
-import {TodoType} from "../../types/todo.type";
+import {Todo} from "../../types/todo";
 
 @Component({
   selector: 'app-todo-list',
   templateUrl: './todo-list.component.html',
   styleUrls: ['./todo-list.component.scss']
 })
 export class TodoListComponent {
-  @Input() todos: TodoType[] = [];
-  @Output() checkedTodoEvent = new EventEmitter<number>();
-  @Output() removeTodoEvent = new EventEmitter<number>();
-  @Input() countAll:number = 0;
-  @Output() checkedAllTodoEvent = new EventEmitter<string>();
-  @Output() editedTodoEvent = new EventEmitter<TodoType>();
+  @Input() todos: Todo[] | null = [];
```

Comment:
```suggestion
  @Input() todos: Todo[] = [];
```

`| null` не стоит добавлять

Reply:
![image](https://github.com/user-attachments/assets/768c98ff-1209-4e17-95b3-6ec1a69391d4)

Ругается на тип

Reply:
добавь `... | async) || []`

Reply:
работает